scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
If we're explaining the breakpoints to a user of Chassis, right now it's:
* ???: 0px–399px
* XS: 400px–767px
* SM: 768px–919px
* etc...

I'm saying that having no name or reification of the "???" viewport size tier is awkward. I think it'd be better to conceptualize the "???" as XS and rename the other tiers accordingly.

(Sidenote: The "???" tier needs no media query since its minimum viewport width is 0px.)